samples/bluetooth/peripheral/prj.conf

```diff
@@ -10,6 +10,7 @@ CONFIG_BT_DIS=y
 CONFIG_BT_ATT_PREPARE_COUNT=5
 CONFIG_BT_BAS=y
 CONFIG_BT_HRS=y
+CONFIG_BT_IAS=y
```

Comment:
This sample isn't using the service for anything. It's OK to add IMO, but the sample should at least register the callbacks and print alert level changes.